editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/ApplicationsTable.tsx

```diff
@@ -0,0 +1,99 @@
+import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import { ApplicationData } from "lib/applications";
+import React from "react";
+
+const StyledTableContainer = styled(TableContainer)(({ theme }) => ({
+  border: `1px solid ${theme.palette.divider}`,
+}));
+
+const StyledTable = styled(Table)({
+  borderCollapse: "collapse",
+});
+
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  border: `1px solid ${theme.palette.divider}`,
+}));
+
+interface Props {
+  applications: ApplicationData[];
+}
+
+function formatDate(date: Date | string) {
+  if (date == "N/A") return date;
+  const formattedDate = new Date(date).toLocaleDateString("en-UK");
+  return formattedDate;
+}
+
+function formatBoolean(state: boolean) {
+  return state ? "Yes" : "No";
+}
+
+function formatAmount(amount: number) {
+  return "£" + (amount / 100).toFixed(2);
+}
```

Comment:
nit: there's some existing helper methods in the `Pay` component like `formattedPriceWithCurrencySymbol()` that might be able to be imported here instead for consistency?

Reply:
I was sure there must be but I hadn't spotted it, I'll get this updated to use the existing helper 👍